packages/manfred2md/src/engine/md-parts/studies-section/studies-section.ejs

```diff
@@ -0,0 +1,24 @@
+---
+
+### EDUCACIÓN
+
+<%_ if (studiesCollection && studiesCollection !== 'undefined') { -%>
+  <%_ for (const study of studiesCollection) { -%>
+#### <%- study.name %>
+
+- **<%- study.description %>**
+
+- **Tipo de Estudio:** <%- study.studyType %>
+
+- **Grado obtenido:** <%- study.degreeAchieved ? "No obtenido" : "Obtenido"%>
+
+- **Fecha de inicio y finalización:** <%- study.startDate %> - <%- study.finishDate %>
```

Comment:
We need to control the finish date because it is possible that the studies have not finished yet, change this please.